core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -104,13 +126,10 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
         .map(n => new ProducerRecord(topic1, partition0, null: Array[Byte], n.toString))
```

Comment:
do you know if we can enable `"resume consumer from committed offset" ignore` again?
I think it was disabled because of failures on Travis. Do you see anything wrong with the test?